fontc/src/args.rs

```diff
@@ -21,12 +21,15 @@ pub struct Args {
     #[arg(long, default_value = "false")]
     pub emit_debug: bool,
 
-    /// Whether to Try Hard(tm) to match fontmake (Python) behavior in cases where there are other options.
-    ///
-    /// See <https://github.com/googlefonts/fontmake-rs/pull/123> for an example of
-    /// where this matters.
+    /// In cases where a source glyph uses a mixture of components and contours, convert
+    /// all the components to contours.
     #[arg(long, default_value = "true")]
-    pub match_legacy: bool,
+    pub prefer_simple_glyphs: bool,
+
+    /// Eliminate component references to other glyphs using components, emitting only
```

Comment:
I like the word "nested" if you can place it somewhere in this docstring

Reply:
on it!